.gitignore

```diff
@@ -1,3 +1,11 @@
-build/
-bsp/uefi/include/bsp/version.h
-bsp/sbl/include/bsp/version.h
+hypervisor/build/
+hypervisor/bsp/uefi/include/bsp/version.h
+hypervisor/bsp/sbl/include/bsp/version.h
+devicemdodel/build/
+devicemodel/include/version.h
+doxygen
```

Comment:
these gitignore items from the acrn-documentation repo need to have ``doc/`` prepended (as you did for the other repos)